pkg/feature/domain/flag_trigger.go

```diff
@@ -92,11 +77,15 @@ func (ft *FlagTrigger) UpdateTriggerUsage() error {
 }
 
 func (ft *FlagTrigger) GenerateToken() error {
+	newTriggerUuid, err := uuid.NewUUID()
+	if err != nil {
+		return err
+	}
 	secret := NewFlagTriggerSecret(
 		ft.GetId(),
```

Comment:
We just need need the UUID. you can delete the `NewFlagTriggerSecret`.

Reply:
You mean we can use `uuid` as the `token` directly?

Reply:
I mean, you don't need to create a struct just for one field.
You can call it directly.

E.g.
```go
h.Write([]byte(newTriggerUuid.String()))
```

Reply:
I understand what you mean. 
We can eliminate the `FlagTriggerSecret` struct and then hash the `UUID` to obtain the token.